frontend/src/pages/Landing/Components/TooltipWithEllipsis.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
+
+import './TooltipWithEllipsis.css';
+
+export default function TooltipWithEllipsis({ collection, collectionTitle }) {
+  const [showTooltip, setShowTooltip] = useState(false);
+  const [cssClasses, setCssClasses] = useState('smart-hub--tooltip-with-ellipsis');
+
+  useEffect(() => {
```

Comment:
I'm not sure cssClasses needs to be `state` of this component. Should be able to just set `cssClasses` as a normal variable (also outside of the `useEffect`).

Reply:
Good catch. Honestly, I think it was left over from an earlier version of this where I had spent all kinds of elaborate effort to use a ref and position the tooltip fixed-ly.